docs/teams.md

```diff
@@ -1 +1,125 @@
+
+### ApplicationTeam
+
+To create an `application_team` for your cluster, you will need to supply a team name, with the options to pass map of labels, map of resource quotas, existing IAM entities (user/roles), and a directory where you may optionally place any policy definitions and generic manifests for the team.
+These manifests will be applied by the platform and will be outside of the team control
+
+**NOTE:** When the manifests are applied, namespaces are not checked. Therefore, you are responsible for namespace settings in the yaml files.
+> As of today (2020-05-01), resource `kubernetes_manifest` can only be used (`terraform plan/apply...`) only after the cluster has been created and the cluster API can be accessed. Read ["Before you use this resource"](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/manifest#before-you-use-this-resource) section for more information.
+To overcome this limitation, you can add/enable `manifests_dir` after you applied and created the cluster first. We are working on a better solution for this.
+
+#### Application Team Example
+
+```hcl
+  # EKS Application Teams
+
+  application_teams = {
+    # First Team
+    team-blue = {
+      "labels" = {
+        "appName"     = "example",
+        "projectName" = "example",
+        "environment" = "example",
+        "domain"      = "example",
+        "uuid"        = "example",
+      }
+      "quota" = {
+        "requests.cpu"    = "1000m",
+        "requests.memory" = "4Gi",
+        "limits.cpu"      = "2000m",
+        "limits.memory"   = "8Gi",
+        "pods"            = "10",
+        "secrets"         = "10",
+        "services"        = "10"
+      }
+      manifests_dir = "./manifests"
+      # Belows are examples of IAM users and roles
+      users = [
+        "arn:aws:iam::123456789012:user/blue-team-user",
+        "arn:aws:iam::123456789012:role/blue-team-sso-iam-role"
+      ]
+    }
+
+    # Second Team
+    team-red = {
+      "labels" = {
+        "appName"     = "example2",
+        "projectName" = "example2",
+      }
+      "quota" = {
+        "requests.cpu"    = "2000m",
+        "requests.memory" = "8Gi",
+        "limits.cpu"      = "4000m",
+        "limits.memory"   = "16Gi",
+        "pods"            = "20",
+        "secrets"         = "20",
+        "services"        = "20"
+      }
+      manifests_dir = "./manifests2"
+      users = [
+
+        "arn:aws:iam::123456789012:role/other-sso-iam-role"
+      ]
+    }
+  }
+```
+
+The `application_teams` will do the following for every provided team:
+
+- Create a namespace
+- Register quotas
+- Register IAM users for cross-account access
+- Create a shared role for cluster access. Alternatively, an existing role can be supplied.
+- Register provided users/role in the `awsAuth` map for `kubectl` and console access to the cluster and namespace.
```

Comment:
"`aws-auth` configmap for..."   -  should probably fix it on the module README as-well

Reply:
Fixed in both files.